aws-organizations-organizationalunit/src/main/java/software/amazon/organizations/organizationalunit/BaseHandlerStd.java

```diff
@@ -132,34 +133,39 @@ public final boolean isRetriableException (Exception e){
     }
 
     public final ProgressEvent<ResourceModel, CallbackContext> handleRetriableException(
-        final OrganizationsRequest organizationsRequest,
-        final ProxyClient<OrganizationsClient> proxyClient,
-        final CallbackContext context,
-        final Logger logger,
-        final Exception e,
-        final ResourceModel model,
-        final Constants.Action actionName,
-        final Constants.Handler handlerName
+            final OrganizationsRequest organizationsRequest,
+            final ProxyClient<OrganizationsClient> proxyClient,
+            final CallbackContext context,
+            final Logger logger,
+            final Exception e,
+            final ResourceModel model,
+            final Constants.Action actionName,
+            final Constants.Handler handlerName
     ) {
-        try {
-            String ouInfo = model.getId() == null ? model.getName() : model.getId();
-            if (actionName != Constants.Action.CREATE_OU) {
-                int currentAttempt = context.getCurrentRetryAttempt(actionName,handlerName);
-                if (currentAttempt < MAX_RETRY_ATTEMPT_FOR_RETRIABLE_EXCEPTION) {
+        String ouInfo = model.getId() == null ? model.getName() : model.getId();
+        if (actionName != Constants.Action.CREATE_OU) {
+            int currentAttempt = context.getCurrentRetryAttempt(actionName, handlerName);
+            if (currentAttempt < MAX_RETRY_ATTEMPT_FOR_RETRIABLE_EXCEPTION) {
+                context.setCurrentRetryAttempt(actionName, handlerName);
+                if (actionName == Constants.Action.DESCRIBE_OU
+                        || actionName == Constants.Action.LIST_PARENTS
```

Comment:
Same comment as above regarding using Handler instead of Action

Reply:
Fixed